biz.ganttproject.core/src/main/java/biz/ganttproject/core/chart/scene/TaskTableSceneBuilder.kt

```diff
@@ -0,0 +1,91 @@
+package biz.ganttproject.core.chart.scene
+
+import biz.ganttproject.core.chart.canvas.Canvas
+import biz.ganttproject.core.chart.canvas.TextMetrics
+import java.awt.Color
+import java.lang.Integer.max
+
+class TaskTableSceneBuilder(
+  private val rows: List<RowContent>,
+  private val input: InputApi,
+  val canvas: Canvas = Canvas()
+) {
+  private val dimensions = calculateDimensions()
```

Comment:
I think dimensions shall be recalculated on every build. And perhaps the build shall just return a new instance of Canvas instead of clearing the old one